apps/server/src/modules/school/controller/school-external-tool.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { ApiTags } from '@nestjs/swagger';
+import { Controller, Get, Param } from '@nestjs/common';
+import { ICurrentUser } from '@shared/domain';
+import { ExternalToolDO } from '@shared/domain/domainobject/external-tool';
+import { Authenticate, CurrentUser } from '../../authentication/decorator/auth.decorator';
+import { SchoolExternalToolListResponse, SchoolParams } from './dto';
+import { SchoolExternalToolUc } from '../uc/school-external-tool.uc';
+import { SchoolExternalToolMapper } from './mapper/school-external-tool.mapper';
+
+@ApiTags('School')
+@Authenticate('jwt')
+@Controller('school')
+export class SchoolExternalToolController {
+	constructor(
+		private readonly schoolExternalToolUc: SchoolExternalToolUc,
+		private readonly schoolExternalToolMapper: SchoolExternalToolMapper
+	) {}
+
+	@Get(':schoolId/available-tools')
```

Comment:
What do you think about :schoolId/tools with query param for available, maybe as default ? This looks not really rest conform